components/layout/layout_impl.rs

```diff
@@ -459,6 +459,17 @@ impl Layout for LayoutThread {
                 .unwrap_or_default(),
         });
 
+        reports.push(Report {
+            path: path![formatted_url, "layout-thread", "stacking_context_tree"],
+            kind: ReportKind::ExplicitJemallocHeapSize,
+            size: self
+                .stacking_context_tree
+                .borrow()
+                .as_ref()
+                .map(|tree| tree.size_of(ops))
+                .unwrap_or_default(),
```

Comment:
I'm pretty sure we can write this as `self.stacking_context_tree.size_of(ops)` instead.

Reply:
The other entries are a bit different because they use `conditional_size_of` for the Arc values but they're behind additional layers of types.